KukaApp/.gitignore

```diff
@@ -61,3 +61,7 @@ buck-out/
 
 # CocoaPods
 /ios/Pods/
+
+
+# Testing
+__tests__/.coverage/
```

Comment:
As a developer I can now do `yarn test --coverage` (again this is just a optional preference that comes with Jest and REALLY helpful for developers like me)